tests/common/plugins/conditional_mark/hooks/pre-commit.py

```diff
@@ -0,0 +1,25 @@
+import re
+import sys
+
+def main():
+    stage_files = sys.argv[1:]
+    for file in stage_files:
+        conditions = []
+        with open(file, 'r') as f:
+            file_contents = f.readlines()
+            if not file_contents:
+                continue
+            for line in file_contents:
+                if re.match('[a-zA-Z]', line):
+                    conditions.append(line.splitlines()[0][:-1])
+            pre = conditions[0]
+            for condition in conditions[1:]:
+                print(condition)
```

Comment:
Will this line generate large amount of logs to console and overwhelm people running git commit if we have big conditional files?

Reply:
I forget to delete this print. It was used for debug.